clr.py

```diff
@@ -327,23 +365,27 @@ def on_batch_end(self, batch, logs=None):
             x = X[idx]
             y = Y[idx]
 
-            values = self.model.evaluate(x, y, batch_size=self.batch_size, verbose=False)
+            values = self.model.evaluate(
+                x, y, batch_size=self.batch_size, verbose=False)
             loss = values[0]
         else:
             loss = logs['loss']
 
         # smooth the loss value and bias correct
-        running_loss = self.loss_smoothing_beta * loss + (1. - self.loss_smoothing_beta) * loss
-        running_loss = running_loss / (1. - self.loss_smoothing_beta ** self.current_batch_)
+        running_loss = self.loss_smoothing_beta * loss + (
+            1. - self.loss_smoothing_beta) * loss
+        running_loss = running_loss / (
+            1. - self.loss_smoothing_beta**self.current_batch_)
 
         # stop logging if loss is too large
         if self.current_batch_ > 1 and self.stopping_criterion_factor is not None and (
-                running_loss > self.stopping_criterion_factor * self.best_loss_):
+                running_loss >
+                self.stopping_criterion_factor * self.best_loss_):
 
             if self.verbose:
-                print(" - LRFinder: Skipping iteration since loss is %d times as large as best loss (%0.4f)" % (
-                    self.stopping_criterion_factor, self.best_loss_
-                ))
+                print(
+                    " - LRFinder: Skipping iteration since loss is %d times as large as best loss (%0.4f)"
```

Comment:
Put the first line on the same level as print. This new line corrects exactly one char spacing.